e2e_samples/mdw_governance/README.md

```diff
@@ -20,9 +22,9 @@ This sample demonstrates how to provision end-to-end modern data warehouse solut
 
 ## Solution Overview
 
-This solution sets up an Azure Data Lake storage account, with two containers: Datalake and Dropzone. The folder structure in datalake is structured to enable data tiering (Bronze, Silver, Gold), hold shared data (Reference) and shared libraries (Sys). Azure Data Factory instance with linked services connecting to the Azure Data Lake, Azure Key Vault and Azure Purview. Application Insights is used for event logging and Office365 API Connection in combination with Logic Apps is used for sending notification emails. A Virtual Network and Private Endpoints are deployed for Data Lake and Key Vault, however the firewall on the services is left open in the initial deployment. To fully secure the solution, the Virtual Network should be peered to an internal corporate network, and firewall closed on those services.
+This solution sets up an Azure Data Lake storage account, with two containers: Datalake and Dropzone. The folder structure in datalake is structured to enable data tiering (Bronze, Silver, Gold), hold shared data (Reference) and shared libraries (Sys). Azure Data Factory instance with linked services connecting to the Azure Data Lake, Azure Key Vault, Azure Databricks and Azure Purview. Application Insights is used for event logging and Office365 API Connection in combination with Logic Apps is used for sending notification emails. A Virtual Network and Private Endpoints are deployed for Data Lake and Key Vault, however the firewall on the services is left open in the initial deployment. Additionally the Databricks virtual network is not peered to the solution's and the databricks public IP is enabled. To fully secure the solution, the Virtual Network should be peered to an internal corporate network, and firewall closed on those services.
```

Comment:
Could you also add a link to the existing single tech sample which demonstrates locking down a Databricks environment: https://github.com/Azure-Samples/modern-data-warehouse-dataops/tree/master/single_tech_samples/databricks/sample2_enterprise_azure_databricks_environment